presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -787,9 +795,10 @@ std::shared_ptr<connector::ConnectorTableHandle> toConnectorTableHandle(
     return std::make_shared<connector::hive::HiveTableHandle>(
         tableHandle.connectorId,
         tableName,
-        true,
+        hiveLayout->pushdownFilterEnabled,
```

Comment:
Should we remove this flag from Velox? It would be nice to support only one code path with pushdown enabled.

Reply:
We don't plan to have pushdown support for slower readers like TEXTFILE at least for this half.  It's still useful to serve as a safeguard to check that tuple domains are empty for these tables:
https://github.com/facebookincubator/velox/blob/9cbeb8b190b9cbcb3f5a7413e78ccaa2620aabf3/velox/connectors/hive/HiveDataSource.cpp#L423